ssc/cmod_irradproc.cpp

```diff
@@ -29,230 +29,264 @@ OF THE USE OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
+        { SSC_INPUT,        SSC_NUMBER,      "lon",                        "Longitude",                        "deg",    "",                      "Irradiance Processor",      "*",                        "",                      "" },
+        { SSC_INPUT,        SSC_NUMBER,      "tz",                         "Time zone",                        "hr",     "",                      "Irradiance Processor",      "*",                        "",                      "" },
+
+        { SSC_INPUT,        SSC_NUMBER,      "sky_model",                  "Tilted surface irradiance model", "0/1/2", "Isotropic,HDKR,Perez",  "Irradiance Processor",      "?=2",                     "INTEGER,MIN=0,MAX=2", ""},
+
+        { SSC_INPUT,        SSC_NUMBER,      "track_mode",                 "Tracking mode",                  "0/1/2",  "Fixed,1Axis,2Axis",     "Irradiance Processor",      "*",                       "MIN=0,MAX=2,INTEGER",                      "" },
+        { SSC_INPUT,        SSC_NUMBER,      "azimuth",                    "Azimuth angle",                  "deg",    "E=90,S=180,W=270",      "Irradiance Processor",      "*",                       "MIN=0,MAX=360",                            "" },
+        { SSC_INPUT,        SSC_NUMBER,      "tilt",                       "Tilt angle",                     "deg",    "H=0,V=90",              "Irradiance Processor",      "?",                       "MIN=0,MAX=90",                             "" },
+        { SSC_INPUT,        SSC_NUMBER,      "rotlim",                     "Rotational limit on tracker",    "deg",    "",                      "Irradiance Processor",      "?=45",                    "MIN=0,MAX=90",                             "" },
+        { SSC_INPUT,        SSC_NUMBER,      "backtrack",                  "Enable backtracking",            "0/1",    "",                      "Irradiance Processor",      "?=0",                    "BOOLEAN",                                  "" },
+        { SSC_INPUT,        SSC_NUMBER,      "gcr",                        "Ground coverage ratio",          "0..1",   "",                      "Irradiance Processor",      "backtrack=1",              "MIN=0,MAX=1",                               "" },
+        { SSC_INPUT,        SSC_NUMBER,      "elevation",                  "Elevation",                "m",                 "",                  "POA Calibrate", "?",           "",                   "" },
+        { SSC_INPUT,        SSC_NUMBER,      "dry_temperature",            "Dry Temperature",                "°C",                 "",      "Irradiance Processor", "?",           "",                   "" },
+        { SSC_INPUT,        SSC_NUMBER,      "pressure",                   "Pressure",                "millibars",                 "",                    "Irradiance Processor", "?",           "",                   "" },
```

Comment:
please align the entries in the table so that it's easy to read (seems to lose alignment with the units)